content/desktop/release-notes.md

```diff
@@ -24,11 +24,60 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/releases.md).
 
+## 4.22.0
+
+{{< release-date date="2023-08-03" >}}
+
+{{< desktop-install all=true version="4.22.0" build_path="/117440/" >}}
+
+### Upgrades
+- [Compose v2.21.0](https://github.com/docker/compose/releases/tag/v2.21.0)
+- [Docker Engine v24.0.6](https://docs.docker.com/engine/release-notes/24.0/#2406)
+- [Docker Scout CLI v0.24.1](https://github.com/docker/scout-cli/releases/tag/v0.24.1). With Docker Scout, you can now:
```

Comment:
Upon reflection, I'm not sure all of this looks right here 🤔 
Let's keep the `[Docker Scout CLI v0.24.1](https://github.com/docker/scout-cli/releases/tag/v0.24.1)` where it is, but more `With Docker Scout, you can now:...` and all the bullet points to below 'Bug fixes and enhancements` section